collections/_article/introducing-reverse-z.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Introducing Reverse Z (AKA I'm sorry for breaking your shader)"
+excerpt: "We are breaking compatibility for some custom shaders. Here is why."
+categories: ["news"]
+author: Clay John
+image: /storage/blog/covers/reverse-z.webp
```

Comment:
The file name should match the article name ideally so we can match them easily.